EventsExpress.Db/Migrations/20201107090613_AddMultiplyEventOwners.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace EventsExpress.Db.Migrations
+{
+    public partial class AddMultiplyEventOwners : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Events_Users_OwnerId",
+                table: "Events");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Events_OwnerId",
+                table: "Events");
+
+            migrationBuilder.DropColumn(
+                name: "OwnerId",
+                table: "Events");
+
+            migrationBuilder.CreateTable(
+                name: "EventOwners",
+                columns: table => new
+                {
+                    UserId = table.Column<Guid>(nullable: false),
+                    EventId = table.Column<Guid>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_EventOwners", x => new { x.EventId, x.UserId });
+                    table.ForeignKey(
+                        name: "FK_EventOwners_Events_EventId",
+                        column: x => x.EventId,
+                        principalTable: "Events",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                    table.ForeignKey(
+                        name: "FK_EventOwners_Users_UserId",
+                        column: x => x.UserId,
+                        principalTable: "Users",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
+            migrationBuilder.CreateIndex(
```

Comment:
Did you regenerate migrations or removed the column reference? Please, regenerate migrations - there should not be such column.